### DISCUSSION TITLE
Customer portal functionnality (login as contact or account) still available ?

### DISCUSSION THREAD

**Discussion by apcloic** [YetiForceCompany/YetiForceCRM#17126] [General] [answered]
Hi,

I'm trying to help a non-profit organization, I need their members to be able to login and then create some kind of "request" (handled by a custom module).

I thought I could use the portal along with a premium paid subscription.

I did some test on Yetiforce official demo portal, I've created a new contact, enable the "portal user" checkbox and make sure permission to send email is enabled too but email is not sent (I suppose it's for spam/security reason). 

Then I've tried to use the Reset Password feature on portal but it returns the following error :
``
User with this email address does not exist
``

Does this functionnality, still available, I mean login as contact / not crm user ?

Regards,

**Comment by rskrzypczak** [comment, marked as answer]
did you assign the contact to the api user?
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/74786ccf-35bb-4a4f-a9c6-85ea11ca5e76)

**Reply by apcloic** [reply]
@rskrzypczak
Thank you, I didn't know it requires an extra stop to assign the contact to API user.
I will try this way. 
Ty
